<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" name="modelo.modelo_tests.TestDeposito-20160214122307" tests="7" time="0.087">
	<testcase classname="modelo.modelo_tests.TestDeposito" name="test1_save_try" time="0.028"/>
	<testcase classname="modelo.modelo_tests.TestDeposito" name="test2_save_except" time="0.008"/>
	<testcase classname="modelo.modelo_tests.TestDeposito" name="test3_find" time="0.015"/>
	<testcase classname="modelo.modelo_tests.TestDeposito" name="test3_notfind" time="0.010"/>
	<testcase classname="modelo.modelo_tests.TestDeposito" name="test4_dlt" time="0.020"/>
	<testcase classname="modelo.modelo_tests.TestDeposito" name="test4_dlt_except" time="0.004"/>
	<testcase classname="modelo.modelo_tests.TestDeposito" name="test5_repr" time="0.002"/>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
